vagrant/provisioners/install-lucee.sh

```diff
@@ -63,7 +63,7 @@ sudo /bin/cp -f /vagrant/configs/lucee-server.xml /opt/lucee/lib/lucee-server/co
 sudo cp /vagrant/configs/server.xml /opt/lucee/tomcat/conf/server.xml
 
 sudo mkdir -p /opt/lucee/config/web
-sudo cp -f /vagrant/configs/web.xml /opt/lucee/tomcat/conf/web.xml
```

Comment:
Actually, just deny this request. The updated libraries don't work with each other so well. I did a fresh install of vagrant with the changes and it's running into issues. Until I have time to sort them out, just cancel the pull.
I did try to add my own web.xml file in there, but it didn't work. If you have the correct one, please do so I can test again.